### DISCUSSION TITLE
icon_set is wrong

### DISCUSSION THREAD

**Discussion by jhonriv** [qax-os/excelize#1933] [Q&A]
Greetings, I have been including new functions in my code and I want to use the conditional format to include icon set indicators, after trying to apply the documentation I see that I have not been able to get on this specific topic, and I have not been able to configure from the properties of ConditionalFormatOptions, reviewing your code, I realized that the parameters I want to configure are not available and also are set directly without the possibility of editing, my question is, is it possible that you can update the library so that you can edit the values of condFmtIconSetPresets? In my case I need to be able to change the cfvo3, the array cfvo3.IconSet.Cfvo:

`
val1, val2, val3, type := (as dynamic values)
`

`
[]*xlsxCfvo{
		{Type: type, Val: val1},
		{Type: type, Val: val2},
		{Type: type, Val: val3},
	}}
`

![image](https://github.com/qax-os/excelize/assets/41612042/899e9bf4-12fe-4cd2-8fe0-8db1d6ac823f)

**Comment by xuri** [comment]
Thanks for your feedback. I'm not sure what you expected. Could you show us a complete, standalone example program or reproducible demo?
